app/schema/changeLog.ts

```diff
@@ -9,7 +9,7 @@ export const changeLogEntrySchema = z.object({
   record_id: z.number(),
   record_type: z.string(),
   record_json: z.object({
-    old: z.record(z.string(), z.unknown()),
+    old: z.record(z.string(), z.unknown()).nullable(),
     new: z.record(z.string(), z.unknown()),
   }),
 });
```

Comment:
i ran into a Zod exception when i connected to the staging DB. we might want to bring this commit upstream.

Reply:
i did this by cherry-picking `0d4611c65acc71b914d4b3ab1b178f38ef136e41` into `nextjs`